packages/components/src/components/post-accordion/post-accordion.tsx

```diff
@@ -52,28 +53,30 @@ export class PostAccordion {
   collapseToggleHandler(event: CustomEvent<boolean>) {
     event.stopPropagation();
```

Comment:
With the event guard in place, I would expect, this is not needed anymore.
Not 100% sure.